otter_welcome_buddy/common/discord_channel.py

```diff
@@ -0,0 +1,25 @@
+import os
+from enum import Enum
+
+from discord.ext.commands import Bot
+from dotenv import load_dotenv
+
+from otter_welcome_buddy.common.utils.types.common import DiscordChannelType
+
+load_dotenv()
+
+
+class DiscordChannel(Enum):
+    """
+    Enum to store Discord channels variables
+    """
+
+    ENGLISH_CHANNEL_ID = int(os.environ["GENERAL_CHANNEL_ID"])
+    NON_ENGLISH_TEACHERS_CHANNEL_ID = int(os.environ["ROOT_CHANNEL_ID"])
```

Comment:
These two variables I think you can move them, I was going to add a file called `settings.py` at the same level than the main, you can create that file and add these over there. Either way, we should revisit these variables later and make them configurable directly from the bot on a future iteration